
geometry_msgs/Pose pose_sync

uint8[] pc_sync



